analyze-ms-stats/calc-manuscript-stats.py

```diff
@@ -0,0 +1,86 @@
+import json
+import subprocess
+import pandas as pd
+import matplotlib
+import argparse
+import multiprocessing
+
+def analyze_commit(commit):
+    """Access files and data in variables.json associated with each commit
+    Accepts commit ID # as string
+    Returns list of 5 statistics"""
+
+    variablesCommand = "git show " + commit + ":./variables.json"
+    variables = json.loads(subprocess.getoutput(variablesCommand))
+
+    date = variables['pandoc']['date-meta']
+    clean_date = variables['manubot']['date']
+    num_authors = len(variables['manubot']['authors'])
+    word_count = variables['manubot']['manuscript_stats']['word_count']
+
+    # Access files and data in references.json associated with each commit
+    referencesCommand = "git show " + commit + ":./references.json"
+    references = json.loads(subprocess.getoutput(referencesCommand))
+    num_ref = len(references)
+
+    return([date, clean_date, num_authors, word_count/1000, num_ref])
+
+def main(args):
+    '''Extract statistics from the output branch log'''
+
+    # Access the variables.json and references.json files associated with each commit and store in dictionary
+    with open(args.commit_list, "r") as commitFile:
+        commits = [c.strip() for c in commitFile.read().splitlines()]
+        pool = multiprocessing.Pool(processes=multiprocessing.cpu_count())
+        commitData = dict(zip(commits, pool.map(analyze_commit, commits)))
+        pool.close()
+        pool.join()
+
+    # Convert dictionary to dataframe
+    growthdata = pd.DataFrame.from_dict(commitData, orient="index",
+                                        columns=["Date", "clean_date", "Authors", "Word Count", "References"])
+    manuscript_stats = growthdata.iloc[0].to_dict()
+    for item in ["Authors", "Word Count", "References"]:
+        manuscript_stats[item] = str(manuscript_stats[item])
+    growthdata = growthdata[::-1]
+    growthdata = growthdata.set_index("Date")
+
+    # Plot the data
+    axes = growthdata.plot(kind='line', linewidth=2, subplots=True)
+    for ax in axes:
+        ax.set_ylabel('Count')
```

Comment:
For the word count, this will need to be `Count (thousands)` or something similar.